serverless/installing_serverless/installing-openshift-serverless.adoc

```diff
@@ -46,11 +46,11 @@ The following limitations apply to all {ServerlessProductName} deployments:
 For more advanced use-cases such as logging or metering on {product-title}, you must deploy more resources. Recommended requirements for such use-cases are 24 CPUs and 96GB of memory.
 
 If you have high availability (HA) enabled on your cluster, this requires between 0.5 - 1.5 cores and between 200MB - 2GB of memory for each replica of the Knative Serving control plane.
-HA is enabled for some Knative Serving components by default. You can disable HA by following the documentation on xref:../../serverless-config-HA.adoc#serverless-config-HA[Configuring High Availability components].
+HA is enabled for some Knative Serving components by default. You can disable HA by following the documentation on xref:../serverless-HA.adoc#serverless-HA[Configuring high availability replicas on {ServerlessProductName}].
```

Comment:
@mburke5678 minor note to remember for peer review. The links should traverse back to the base and then out again. Fixed via: https://github.com/openshift/openshift-docs/pull/23109